src/components/cards/Port.astro

```diff
@@ -17,7 +17,7 @@ const { title, link, categories, color } = Astro.props;
 
 <a href={link} class="port-card">
   <div class="port-header">
-    <p class="port-name">{title}</p>
+    <p class="port-name">{Array.isArray(title) ? title.join(", ") : title}</p>
```

Comment:
I believe we separated with a slash over at userstyles.

Reply:
Not on the rich display.
![image](https://github.com/catppuccin/website/assets/71222764/7cbe329f-36de-41f7-be27-4f7a1f77df0c)

Reply:
Right. Is that something we should change potentially? The linter is requiring titles to be using slashes as separators for the title, and commas/ands for the descriptions.

Reply:
> Right. Is that something we should change potentially? The linter is requiring titles to be using slashes as separators for the title, and commas/ands for the descriptions.

Probably best to continue this discussion on the userstyles repo or on the discord server.